platform/README.mdx

```diff
@@ -56,7 +56,7 @@ This process, known as cryptographic device attestation, forms the foundation fo
 
 # How can you use Smallstep?
 
-The Smallstep Agent is the vehicle through which Smallstep delivers cryptographically attested device identity to your organisation. It is the recommended way to identify devices and get client certificates to devices (Windows, Linux, Mac OS) for Enterprise Wi-Fi, VPN, HTTP/3 proxies, or web applications. 
+The Smallstep Agent is the vehicle through which Smallstep delivers cryptographically attested device identity to your organisation. It is the recommended way to identify devices and get client certificates to devices (Windows, Linux, Mac OS, ChromeOS) for Enterprise Wi-Fi, VPN, HTTP/3 proxies, or web applications. 

```

Comment:
```suggestion
The Smallstep Agent is the vehicle through which Smallstep delivers cryptographically attested device identity to your organisation. It is the recommended way to identify devices and get client certificates to devices (Windows, Linux, macOS, ChromeOS) for Enterprise Wi-Fi, VPN, HTTP/3 proxies, or web applications. 

```

Reply:
Also curious why it's specifically HTTP/3. Just because it's novel? Or as a general term for network relay, which happens to be HTTP/3 (generally, but can be HTTP/2).

Reply:
I think we were using "HTTP/3 proxy" as a shorthand for MASQUE?